File: python/svix/models/list_response_operational_webhook_endpoint_out.py
# this file is @generated
import typing as t

from .common import BaseModel
from .operational_webhook_endpoint_out import OperationalWebhookEndpointOut


class ListResponseOperationalWebhookEndpointOut(BaseModel):
    data: t.List[OperationalWebhookEndpointOut]

    done: bool

    iterator: t.Optional[str]

    prev_iterator: t.Optional[str] = None

File: csharp/Svix/Models/ListResponseOperationalWebhookEndpointOut.cs
// this file is @generated
using System.Text.Json.Serialization;

namespace Svix.Models
{
    public class ListResponseOperationalWebhookEndpointOut(
        string? iterator,
        bool done,
        List<OperationalWebhookEndpointOut> data,
        string? prevIterator = null
    ) : BaseModel
    {
        [JsonPropertyName("data")]
        public List<OperationalWebhookEndpointOut> Data { get; set; } = data;

        [JsonPropertyName("done")]
        public bool Done { get; set; } = done;

        [JsonPropertyName("iterator")]
        public string? Iterator { get; set; } = iterator;

        [JsonPropertyName("prevIterator")]
        public string? PrevIterator { get; set; } = prevIterator;
    }
}
